process_basic.go

```diff
@@ -48,7 +48,7 @@ func newBasicProcess(ctx context.Context, opts *CreateOptions) (Process, error)
 		p.Tag(t)
 	}
 
-	p.RegisterTrigger(ctx, makeOptionsCloseTrigger())
+	_ = p.RegisterTrigger(ctx, makeOptionsCloseTrigger())
```

Comment:
should we error if this errors?

Reply:
Yes it should error. I added an error check.